tox.ini

```diff
@@ -104,7 +106,9 @@ deps =
     -e .
     -e testing
 commands =
-    pytest -n auto --ignore={[vars]tst_path}smoke -v --tb native \
+    pytest -n auto --ignore={[vars]tst_path}smoke \
+        --ignore={[vars]tst_path}benchmark --ignore={[vars]testing_tst_path}benchmark \
```

Comment:
Can we use markers instead?

The magic paths are not very ergonomic, e.g. not easy to copy-paste into the terminal.

Reply:
That's true, but we are using them extensively throughout tox.ini already. I feel like it's better to remain consistent.

I wouldn't have any objection to a separate PR that removed them entirely, though. I don't think they add much value, and using explicit paths (for e.g. PYTHONPATH) and markers (for e.g. skipping types of test) could be a nice improvement.